lib/membrane/video_compositor/scene/transformation.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Membrane.VideoCompositor.Scene.Transformation do
+  @moduledoc """
+  Module representing single input, single output transformations of objects.
+
+  Transformations are single frame input - single frame output objects.
+  Transformations can change frame resolution. Cropping, CornersRounding, ColorFiler,
+  RollToBall, etc. can be implemented as transformations.
+  """
```

Comment:
Using `object` here it might be a little bit confusing since `Transformations` are not `Scene` objects.
```suggestion
defmodule Membrane.VideoCompositor.Scene.Transformation do
  @moduledoc """
  Module representing single-input, single-output frames processing graph nodes
  that apply a single effect on the input frame (e.g. crop them or round corners).

  Transformations are single frame input - single frame output nodes of the processing 
  graph.
  Transformations can change frame resolution. Cropping, CornersRounding, ColorFiler,
  RollToBall, etc. can be implemented as transformations.
  """
```